packages/ckeditor5-dev-env/tests/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,304 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { expect } = require( 'chai' );
+const fs = require( 'fs' );
+const mockFs = require( 'mock-fs' );
+const sinon = require( 'sinon' );
+
+describe( 'updatePackageVersions()', () => {
```

Comment:
Thanks for adding some tests for this new util 👍🏻 

It would be great if you could increase the coverage, as it currently is less than 50% for this new util. [Here](https://coveralls.io/builds/45248775/source?filename=packages%2Fckeditor5-dev-env%2Flib%2Frelease-tools%2Ftasks%2Fupdate-package-versions.js) you can check the not covered lines (i.e. the `--dry-run` flag with diff formatting is not covered at all).